src/models/GoalItem.ts

```diff
@@ -1,9 +1,15 @@
+enum Repeat{
+    'Once',
+    'Daily'
+}
+
 export interface GoalItem{
     id?: number;
     title: string;
     duration: string;
```

Comment:
For now we assume all durations are in unit hours for the whole app.
If we do start using a unit specifier later - this will be in a separate field - and probably at calendar level so it doesn't have to be repeated in each goal.

For this reason, if you agree, please change duration to int without specifying the unit.

Reply:
okay.